### DISCUSSION TITLE
Raspberry Pi 4 stand-by state

### DISCUSSION THREAD

**Discussion by kaduleite** [mvp/uhubctl#333] [Q&A] [answered]
For default, when you shutdown raspberry pi 4, the USB ports remains powered. If I disable them using 'uhubctl' and then shutdown it, the power stills off while the power still connected to pi 4?

**Comment by mvp** [comment, marked as answer]
From what I heard, answer is yes, turning power with uhubctl before shutdown should keep it off. But please try it yourself. Thanks!
